code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -377,7 +377,7 @@
 	caliber = "a762"
 	matter = list(DEFAULT_WALL_MATERIAL = 4000)
 	ammo_type = /obj/item/ammo_casing/a762
-	max_ammo = 20
+	max_ammo = 10

```

Comment:
There already exists a 10 round magazine for 762 rounds. This change is redundant.